config_parameters.py

```diff
@@ -0,0 +1,11 @@
+conversion_dict = {("l", "m3"): 1000,
```

Comment:
Something that may make this easier to read/understand is perhaps a comment on each line (or per two lines given there are reverse instances in this dict) that say what each represents.